### PR TITLE
Milestone 03: React Capstone Project

### PR DESCRIPTION
**Pull request by myaserkhan**
### Hi there!

**In this project, I built a mobile web application to check a list of metrics (numeric values) using of React and Redux. An API is selected that provides numeric data about a topic and then build the web app around it. The web app will have several pages:**

- [x] A home page with a list of items that could be filtered by some parameters.
- [x] Another page for the item details

Live Demo 🔗

Check the live demo [Netlify👈](https://teal-travesseiro-0a5b57.netlify.app/)

Project Documentation 📄

Here is the presentation [video](https://www.loom.com/share/597d886f55ba44cdbef8e2fe48358ac6)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Give a ⭐️ if you like this project!
 
-### Analyzing the Bundle Size
+## Acknowledgments
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (KDlamini) [inline README.md]: Kindly make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  return (
+    <>
+      <Header />
+      <MainPreview />
+      <h3>POKEMON BY ID</h3>
+      <section className="list">
+        {pokemons.map((pokemon) => (<Preview key={pokemon.id} pokemon={pokemon} />))}
+      </section>
```
Comment (KDlamini) [inline src/components/Home.js:24]: Kindly make sure that when the Pokemons are displayed on the homepage they have alternating background colors (one lighter background and the other darker background) as per design specifications.

image

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @myaserkhan 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a metrics web app using React and API integration. 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR title and description
✔️  Components, events, actions, and state management well-implemented 
✔️  Components are well-tested
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
### Hello @myaserkhan 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------

**Pull request merged**